### PR TITLE
Document TOC Project Representative Elections 2023

### PR DESCRIPTION
**Pull request by fdegir**
This pull request documents the process to follow together with the timeline for the TOC Project Representative Elections for 2023.

Please note the timeline as well as the upcoming steps in the process as certain artifacts become available on correponding dates.

### CODE REVIEW

**Comment by fdegir** [comment]
@cdfoundation/toc @cdfoundation/toc-contributors Please take a look at this pull request as this starts the CDF TOC Project Representatives Election for 2023 and let us know of your comments.

File: elections/2023/README.md [hunk 1]
```diff
@@ -69,4 +69,66 @@ Elections will be held using time-limited, [ranked voting](https://www.opavote.c
...
+| June 5  | Project Nominations close         | Last date for projects to nominate their candidates
+| June 8  | Candidates announced              | Nominated candidates are announced
+| June 12 | Voting opens                      | Ballots are sent to eligible voters
+| June 26 | Voting closes                     | Last day to submit a ballot
+| June 28 | Announcement                      | Successful project representatives announced
+| July 1  | Term Begins                       | New project representatives beging their term
+
+### Candidates Eligibility
+
+Eeach CDF project nominates one (1) representative to run for a project seat.
```
Comment (e-backmark-ericsson) [inline elections/2023/README.md:92]:
```suggestion
Each CDF project nominates one (1) representative to run for a project seat.
```

File: elections/2023/README.md [hunk 2]
```diff
@@ -69,4 +69,66 @@ Elections will be held using time-limited, [ranked voting](https://www.opavote.c
...
+Four (4) CDF Elected Project Representative Seats are up for election for two (2) years term, 1 July 2023 to 30 June 2025.
+
+### Timeline
+
+| Date    | Event                             | Details
+| ------- | --------------------------------- | -------
+| May 4   | Election Process starts           | CDF TOC Project Representatives are notified regarding the start of election process
+| May 15  | List of eligible voters published | List of eligible voters is determined using LFX Insights and announced
+| May 15  | Voter Registration opens          | Opening date to register to vote
+| May 22  | Project Nominations open          | Start of projects nominating their candidates in GitHub CDF Foundation Repo
```
Comment (e-backmark-ericsson) [inline elections/2023/README.md:81]:
```suggestion
| May 22  | Project Nominations open          | Start of projects nominating their candidates in GitHub CDF 'foundation' Repo
```

File: elections/2023/README.md [hunk 3]
```diff
@@ -69,4 +69,66 @@ Elections will be held using time-limited, [ranked voting](https://www.opavote.c
...
+* good communication skills with experience of participating in open source communities
+* operate neutrally in discussions and put the goals and success of CDF in balance with corporate objectives or any particular project in CDF
+
+Each project is free to choose its candidate in whichever manner fits best for their community.
+
+### Candidate Nominations
+
+[CDF TOC Project Representatives](https://github.com/cdfoundation/toc/blob/main/CONTRIBUTORS.md#project-representatives) will be contacted to nominate their candidate.
+Additionally, the start of the election process will be announced on corresponding maillists (e.g., cdf-toc) as well as CDF Slack.
+The nominations must be made under corresponding GitHub Issue in the [CDF GitHub Foundation repository](https://github.com/cdfoundation/foundation/issues).
```
Comment (e-backmark-ericsson) [inline elections/2023/README.md:106]:
```suggestion
The nominations must be made under corresponding GitHub issue in the [CDF foundation repository](https://github.com/cdfoundation/foundation/issues).
```

File: elections/2023/README.md [hunk 4]
```diff
@@ -69,4 +69,66 @@ Elections will be held using time-limited, [ranked voting](https://www.opavote.c
...
+  * This is to ensure we have voters whose contributions may not be captured through the above methodology. Individuals are able to submit voter form on behalf of other individuals.
+
+The list of eligible voters will be published in **voters.md** file on May 15, 2023.
+
+### Voting Process
+
+Individuals listed in **voters.md** and individuals who are accepted as voters by the election officers will receive a ballot via email.
+If you are not on that list you can use [the voter registration form](https://docs.google.com/forms/d/e/1FAIpQLSdsxCadKwauQ4EHjaNihGtVQJQ027ECsXBZACU7WqXm4dBMCw/viewform) to ask to participate in the election.
+
+Elections will be held using time-limited, ranked voting via OPA Vote.
```
Comment (e-backmark-ericsson) [inline elections/2023/README.md:125]: Duplicated sentence
```suggestion
```

File: elections/2023/README.md [hunk 5]
```diff
@@ -69,4 +69,66 @@ Elections will be held using time-limited, [ranked voting](https://www.opavote.c
...
+* commit that they have the available bandwidth to make the time to invest in the CDF TOC
+* technical background with experience of continuous delivery and open source technologies
+* good communication skills with experience of participating in open source communities
+* operate neutrally in discussions and put the goals and success of CDF in balance with corporate objectives or any particular project in CDF
+
+Each project is free to choose its candidate in whichever manner fits best for their community.
+
+### Candidate Nominations
+
+[CDF TOC Project Representatives](https://github.com/cdfoundation/toc/blob/main/CONTRIBUTORS.md#project-representatives) will be contacted to nominate their candidate.
```
[the remark this answers is not in the document]
Comment (fdegir) [reply elections/2023/README.md:104]: We emailed all project representatives listed here: https://github.com/cdfoundation/toc/blob/main/CONTRIBUTORS.md#project-representatives

File: elections/2023/README.md (hunk 4 above)
Comment (sbtaylor15) [reply elections/2023/README.md:125]: @fdegir have you sent out the opavote.com link via email?  I can't find one at this time.

Comment (fdegir) [reply elections/2023/README.md:125]: @sbtaylor15 The OpaVote link will be sent to eligible voters when the voting starts which will happen on June 12, 2023 once the nomination period is over.

Please see the timeline here: https://github.com/cdfoundation/toc/blob/02a57fd16166f310ea163d72362f4669947d1872/elections/2023/README.md#timeline-1

**Comment by fdegir** [comment]
> Is there a form for voter registration (individuals who may not be making pull requests)? I have it on my calendar to send the form link out today.

@TracyRagan The eligible voter list based on LFX Insights data will be published later today as part of this pull request and then the voter registration form will be enabled for voter registrations.

File: elections/2023/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (e-backmark-ericsson) [reply elections/2023/README.md:104]: Ok. I see. My comment can be ignored then.

File: elections/2023/README.md [hunk 6]
```diff
@@ -69,4 +69,66 @@ Elections will be held using time-limited, [ranked voting](https://www.opavote.c
...
+### Voter Eligibility
+
+Eligible voters will be
+
+* people who had 10 or more commits to any CDF project over the past year (prior to May 1 2023), via data in LFX Insights.
+  * If people don't have LFid, they won't be able to vote. The GitHub handle must be associated with their preferred email address. (Please see [here](https://cd.foundation/blog/staff/2022/11/08/update-your-lfid-for-lfx/) to learn how to do this.)
+* people who have submitted [the voter registration form](https://docs.google.com/forms/d/e/1FAIpQLSdsxCadKwauQ4EHjaNihGtVQJQ027ECsXBZACU7WqXm4dBMCw/viewform) and are accepted by the election officers.
+  * This is to ensure we have voters whose contributions may not be captured through the above methodology. Individuals are able to submit voter form on behalf of other individuals.
+
+The list of eligible voters will be published in **voters.md** file on May 15, 2023.
```
Comment (e-backmark-ericsson) [inline elections/2023/README.md:118]: Is the voters.md file published now? If so, where?
Should it be linked from here?

File: elections/2023/README.md (hunk 2 above)
Comment (fdegir) [reply elections/2023/README.md]: Fixed in the new commit.

File: elections/2023/README.md (hunk 3 above)
Comment (fdegir) [reply elections/2023/README.md]: Fixed in the new commit.

File: elections/2023/README.md (hunk 6 above)
Comment (fdegir) [reply elections/2023/README.md]: The list of eligible voters is now published and available in voters.md file.

File: elections/2023/README.md (hunk 1 above)
Comment (e-backmark-ericsson) [reply elections/2023/README.md]: Corrected in last commit

**Comment by fdegir** [comment]
@e-backmark-ericsson My first comment would be to keep this PR specific to the elections and work out the details for the TOC contributors separately since [the document describing the TOC contributors](https://github.com/cdfoundation/toc/blob/main/CONTRIBUTING.md) is still in draft state.

When it comes to overlaps between the TOC Elected Project Representatives and Project Representatives in the TOC Contributors document -  I believe @afrittoli worked with the projects to identify who could participate in TOC to ensure the projects are aware of and possibility to contribute to the TOC matters.

**Comment by e-backmark-ericsson** [comment]
Ok. I'm satisfied with those comments, and look forward for some clarifications in coming updates of CONTRIBUTING.md outside this PR.

**Review by e-backmark-ericsson — APPROVED** [review]

**Review by afrittoli — APPROVED** [review]
Thanks @fdegir, this lgtm. I will bring it up at today's TOC meeting so we can agree whether it's ok to merge it.

**Review by afrittoli — APPROVED** [review]
Thanks @fdegir, this lgtm. I will bring it up at today's TOC meeting so we can agree whether it's ok to merge it.

**Review by sbtaylor15 — APPROVED** [review]
/lgtm

**Review by oleg-nenashev — APPROVED** [review]

**Review by McGon-Fid — APPROVED** [review]
LGTM too

**Pull request merged**